packages/driver/src/cypress/mocha.js

```diff
@@ -18,6 +18,61 @@ const runnableResetTimeout = Runnable.prototype.resetTimeout
 delete window.mocha
 delete window.Mocha
 
+function overloadMochaFnForConfig (fnName, specWindow) {
+  const _fn = specWindow[fnName]
+
+  const fnType = fnName === 'it' ? 'Test' : 'Suite'
```

Comment:
Do we have tests that handle when people define `xit`? Cause I've seen people define this to skip tests.  

Also, isn't `specify` a valid name for a test function (alias of `it`)? Are there tests for that? https://mochajs.org/#bdd

Reply:
if people define xit it will work assuming they're doing `const xit = it.skip`

Good find with missing 'specify'

Reply:
No, they just use `xit` outright - this works today. 

```
describe("Test", () => {
  it("visit", () => {
    cy.visit("https://example.cypress.io")
  })

  xit("skip visit", () => {
    cy.visit("https://example.cypress.io")
  })
})
```

<img width="504" alt="Screen Shot 2020-04-09 at 3 06 54 PM" src="https://user-images.githubusercontent.com/1271364/78875385-d0bb8780-7a73-11ea-8bdc-4219613e97c4.png">

Reply:
fixed